waspc/src/Wasp/Project/Common.hs

```diff
@@ -86,8 +87,8 @@ tsconfigInWaspProjectDir = [relfile|tsconfig.json|]
 
 findFileInWaspProjectDir ::
   Path' Abs (Dir WaspProjectDir) ->
-  Path' (Rel WaspProjectDir) File' ->
-  IO (Maybe (Path' Abs File'))
+  Path' (Rel WaspProjectDir) (File f) ->
```

Comment:
Nice catch!

Reply:
I also made this change in #2297, I didn't get this positive feedback 😢 https://github.com/wasp-lang/wasp/pull/2297/files#diff-7e31f975a23c9bb416d52191e3af1ea72349963d568ab02226b02d51a7110fb8R99

Reply:
> Nice catch!

It wasn't a catch, my code wasn't compiling so I had to change it :smile: 

> I also made this change in https://github.com/wasp-lang/wasp/pull/2297, I didn't get this positive feedback 😢

Hahaha